docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3052,6 +3052,11 @@ function moveFromDevice() {
   });
 }
 
+function isCurrentUser(member) {
+  const meeting = getCurrentMeeting();
+  return meeting.selfId === member.id
+}
+
```

Comment:
If I get this correctly, this function is checking whether the user is self in the meeting, right?

Reply:
Can be, thanks, I'm good with such naming as well